app/controllers/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
recommend handle incase not found user (@user = nil)
```suggestion
    @user = User.find_by id: params[:id]
    return if @user
    
    flash[:danger] = .... # message not found here
    redirect_to ... # may be using root_path
```